README.md

```diff
@@ -80,6 +80,10 @@ To customize your booru, find and replace all occurences of the following words
 - Rule names that are selectable when reporting violations can be adjusted in `lib/philomena_web/views/report_view.ex`
 - In `config/config.exs` adjust your own `password_pepper`, `otp_secret_key` and `tumblr_api_key`
 - Predefined forum sections can be changed in `priv/repo/seeds.json` in the forums section
+- The baseUrl for image requests can be changed suing the `image_url_root` variable in `config/config.exs`. This can be used to cache image requests using a CDN. The CDN will have to redirect the request to the `/img` endpoint. More Details under `image_url_root` in this document
```

Comment:
typo `suing`. Also, please remove duplicate content. Please either use the subitem xor the separate section.